keyboards/mechlovin/adelais/info.json

```diff
@@ -0,0 +1,80 @@
+{   "keyboard_name": "Adelais",
+    "url": "",
+    "maintainer": "Team Mechlovin",
+    "width": 18,
+    "height": 6,
+    "layouts": {
+        "LAYOUT": {
+            "key_count": 66,
+            "layout": [
```

Comment:
This tree is a mess, but it'll be easier for me to debug if this gets merged as is.

![image](https://user-images.githubusercontent.com/18669334/81251051-46fdcc00-8fd7-11ea-872e-26512fe476aa.png)

Reply:
I have just discovered, my layout is similar to the @Maartenwut  pcb layout.
Hello @Maartenwut. Can I use the info.json file? I'll appreciate that. :D

Reply:
Resolved.

![image](https://user-images.githubusercontent.com/18669334/81440085-5179ac00-9124-11ea-93ef-91a0e4e619c7.png)